### PR TITLE
Updated readme examples to es6

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,61 +40,62 @@ distributions with the [releases][] (but please host your own copy).
...
-    type: 'record',
-    fields: [
-      {name: 'kind', type: {name: 'Kind', type: 'enum', symbols: ['CAT', 'DOG']}},
-      {name: 'name', type: 'string'}
-    ]
-  });
-  var buf = type.toBuffer({kind: 'CAT', name: 'Albert'}); // Encoded buffer.
-  var val = type.fromBuffer(buf); // {kind: 'CAT', name: 'Albert'}
-  ```
+```javascript
```
Comment (mtth) [inline README.md]: Can you restore the indent?

File: README.md [hunk 2]
```diff
@@ -40,61 +40,62 @@ distributions with the [releases][] (but please host your own copy).
...
   container file:
 
-  ```javascript
-  avro.createFileDecoder('./values.avro')
-    .on('metadata', function (type) { /* `type` is the writer's type. */ })
-    .on('data', function (val) { /* Do something with the decoded value. */ });
-  ```
+```javascript
+avro.createFileDecoder('./values.avro')
+  .on('metadata', type => { /* `type` is the writer's type. */ })
```
Comment (mtth) [inline README.md]: `(type)`. (Same below.)

File: README.md [hunk 3]
```diff
@@ -40,61 +40,62 @@ distributions with the [releases][] (but please host your own copy).
...
+```javascript
+avro.assemble('./Ping.avdl', (err, attrs) => {
+  // Generate the protocol and attach a handler for `ping` messages:
+  const protocol = avro.parse(attrs)
+    .on('ping', (req, ee, cb) => {
+      cb(null, 'pong');
+    });
+  // Respond on any incoming connection:
+  require('net').createServer()
+    .on('connection', con => { protocol.createListener(con); })
```
Comment (mtth) [inline README.md]: Since you've split the callback above over multiple lines, can you split this one as well? (And wrap `con` inside parens.)

**Comment by mtth** [comment]
Thanks for the pull request! I added a few comments inline, should be good after. (Don't worry about the unrelated CI failure.)

**Comment by amilajack** [comment]
@mtth Fixed

File: README.md (hunk 2 above)
Comment (mtth) [inline README.md]: You didn't address this.

File: README.md (hunk 3 above)
Comment (mtth) [inline README.md]: And this.

**Comment by mtth** [comment]
Thanks!

**Pull request merged**